src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1658,76 +1657,51 @@ private void MakeSeparatorList(ReadOnlySpan<char> separators, ref ValueListBuild
 
         private void MakeSeparatorListVectorized(ref ValueListBuilder<int> sepListBuilder, char c, char c2, char c3)
         {
-            // Redundant test so we won't prejit remainder of this method
-            // on platforms without SSE.
-            if (!Sse41.IsSupported)
+            if (!Vector128.IsHardwareAccelerated)
```

Comment:
Please use the comment from the previous version (left side of comparison) to make it clear that this check is needed to avoid prejit.
Otherwise a `Debug.Assert(Vector128.IsHardwareAccelerated)` could do it too.

Reply:
I'll reintroduce the comment with a small text change since it's not limited to SSE anymore